### PR TITLE
created base html file and files for js and css

### PR DESCRIPTION
**Pull request by baspeirs**
added html skeleton

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (rtorres61595) [inline index.html]: difbgjerbgf

**Review by baspeirs — COMMENTED** [review]
All changes look good and needed elements have id names.

**Comment by baspeirs** [comment]
Looks good so far, ID's set and we are good to start with the script and the styling

**Review by rtorres61595 — COMMENTED** [review]
test

**Review by seebell — APPROVED** [review]
testing

**Pull request merged**